apps/designer-standalone/src/app/AzureLogicAppsDesigner/Services/WorkflowAndArtifacts.tsx

```diff
@@ -120,7 +119,7 @@ export const listCallbackUrl = async (
 
   return {
     method: callbackInfo.method,
-    value: callbackUri.toString(),
+    value: decodeURI(callbackUri.toString()),
```

Comment:
I'm curious why we need to decode it here? The fact that we want to decode it to get the correct URL means to me that it was double encoded before. Can we find out where it was double encoded?